mediathread/settings_shared.py

```diff
@@ -318,6 +318,8 @@ def default_url_processor(url, label=None, request=None):
     'embed_tool_id': 'mediathread',
 }
 
+SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTOCOL', 'https')
```

Comment:
Oh, good eye there. The salted nginx.conf for Meth (and our other apps) shows X-Forwarded-Protocol being used. I'll do a little digging tomorrow to see if one is preferred over the other.